### PR TITLE
[CS] player turns

### PR DESCRIPTION
**Pull request by tomaas-zeman**
cs-12
cs-13
cs-14

### CODE REVIEW

File: translations/player_turns.tex/cs.po [hunk 1]
```diff
@@ -102,17 +114,17 @@ msgid ""
...
-"  \\item Move a Hero 1 Field in any direction.\n"
-"  \\item \\pagelink{Categories}{Revisit} a Field where your Hero is in.\n"
-"  \\item \\pagelink{Timelimit}{Continue Combat} against Neutral Units for 1 additional Combat Round.\n"
-"  \\item \\pagelink{Placing}{Discover a face down Map Tile} if your Hero is on a Field next to that Tile.\n"
-"  \\item Place a new Map Tile from your pool of Far (II–III) Map Tiles.\n"
+"  \\item Přesun Hrdiny o 1 pole v libovolném směru.\n"
+"  \\item \\pagelink{Categories}{Znovu Navštívit} pole, ve kterém se nachází váš hrdina.\n"
+"  \\item \\pagelink{Timelimit}{Pokračovat v boji} proti Neutrálním Jednotkám další 1 bojové kolo.\n"
+"  \\item \\pagelink{Placing}{Odkrýt Dílek Mapy položený lícem dolů}, pokud je váš Hrdina na poli vedle tohoto Dílku.\n"
+"  \\item Umístit nový Dílek Mapy z vaší zásoby Vzdálených (II-III) Dílků Mapy.\n"
```
Comment (qwrtln) [inline translations/player_turns.tex/cs.po:127]: Careful, wrong place

Comment (qwrtln) [reply translations/player_turns.tex/cs.po:127]: Oh, sorry, I had a wrong impression of the diff.

File: translations/player_turns.tex/cs.po [hunk 2]
```diff
@@ -114,13 +114,14 @@ msgid ""
 "\n"
 msgstr ""
 "\\vspace*{\\lang_header_adjustment}\n"
+"\\iftoggle{printable}{\\vspace*{\\lang_header_adjustment}}{}\n"
```
Comment (qwrtln) [inline translations/player_turns.tex/cs.po:117]: Why? Don't you want it correctly spaced at all times?

Comment (tomaas-zeman) [reply translations/player_turns.tex/cs.po:117]: This is just a workaround for printable mode where I had to decrease the space between sections a bit more to make the text fit to a single page. This was more to keep the aesthetics of the digital version than anything else

Comment (qwrtln) [reply translations/player_turns.tex/cs.po:117]: Exactly, shouldn't the aesthetics be kept for both versions? If the text is too long, you shorten it, not manipulate spacing.

**Review by qwrtln — APPROVED** [review]

File: translations/player_turns.tex/cs.po (hunk 2 above)
Comment (DarthGandalf) [reply translations/player_turns.tex/cs.po]: I haven't look very closely at the source of other PRs, mostly at the pictures attached, you didn't do similar hacks there?

**Pull request merged**